src/Cli/Commands/ValidateOptions.cs

```diff
@@ -26,7 +26,7 @@ public ValidateOptions(string config)
         /// </summary>
         public void Handler(ILogger logger, FileSystemRuntimeConfigLoader loader, IFileSystem fileSystem)
         {
-            logger.LogInformation("{productName} {version}", PRODUCT_NAME, ProductInfo.GetProductVersion());
+            logger.LogInformation("{productName} {version}", PRODUCT_NAME, ProductInfo.GetProductVersion(includeCommitHash: true));
```

Comment:
we dont need the hash even on the first line of the CLI command, so we can at least avoid the hash here by not changing this file altogether.

Reply:
Thanks for pointing out. I too eagerly reverted to using commit hash. Commit hash will only be printed when: CLI usage with `--help`, `--version` , or unknown verbs/options are used.
Tests and CLI options parsing now properly only prints version without commit hash where possible.